dev/run-tests-jenkins.py

```diff
@@ -232,6 +234,10 @@ def main():
     result_message = github_message('has finished')
     result_message += '\n' + test_result_note + '\n'
     result_message += '\n'.join(pr_check_results)
+    result_message += '\n' + ' * This patch tested by `%s` with `%s` and `%s`.' % (
```

Comment:
If you want to keep these changes but revert the others, I think this could be fine.